bitnami/concourse/templates/worker/service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.worker.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "common.names.fullname" . }}-worker
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: worker
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ## We do NOT expose any port as workers will forward connections with the ATC through a TSA reverse-tunnel
+  ## ref: https://concourse-ci.org/architecture.html#architecture-worker
```

Comment:
This link does not work

Reply:
I'm not sure I properly understand the role of this headless service.

Reply:
Where is the reverse tunnel?

Reply:
I update this part to 
```yaml
  ## We do NOT expose any port as workers will forward connections with the ATC through a TSA
  ## ref: https://concourse-ci.org/internals.html#component-tsa
```